pkg/cmd/generate/util.go

```diff
@@ -159,6 +159,7 @@ const header = `# --------------------------------------------------------------
 func writeFile(filePath string, content []byte) error {
 	// https://github.com/kubernetes/kubernetes/issues/67610
 	contentString := strings.ReplaceAll(string(content), "\n  creationTimestamp: null", "")
+	contentString = strings.ReplaceAll(contentString, "\n      creationTimestamp: null", " {}")
```

Comment:
Need to set to empty object to fix issue with ArgoCD synch loop created by TierTemplates do to spec.template.matadata see the fix in infra deployment here: https://github.com/redhat-appstudio/infra-deployments/pull/4120. The requirement is that all TierTemplates in source will start with empty object {} and then we need to remove the creationTimeStap set to null. If there is a better way to do this let me know.

Reply:
let's fix this empty metadata as a separate PR as this won't work if the spec.template.metadata.name is set (for example in our e2e tests).
Let's focus on the empty creationTimestamp for now

Reply:
see my new approach in utils.go, seems simpler to me